ui/lib/pki/addon/components/page/pki-tidy-form.hbs

```diff
@@ -0,0 +1,72 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      <Icon @name="pki" @size="24" class="has-text-grey-light" />
+      Tidy
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<hr class="is-marginless has-background-gray-200" />
+<p class="has-top-margin-m has-bottom-margin-l">Tidying cleans up the storage backend and/or CRL by removing certificates
+  that have expired and are past a certain buffer period beyond their expiration time.</p>
+
+<form class="has-bottom-margin-s" {{on "submit" (perform this.save)}}>
+  <div class="has-bottom-margin-s">
+    <Input
+      @type="checkbox"
```

Comment:
For this element to be associated to the correct label do we need to add `id="tidy-certificate-store"`here?

Reply:
Interesting - I didn't know that an element needs to have an `id` in order to be associated with the input field! It makes sense for readability! I'll add!